index.js

```diff
@@ -0,0 +1,52 @@
+let addIngredient = () => {
+    let ul = document.getElementById("ingredients");
+    let random = document.createElement("li");
+    let ingredient = document.getElementById("newIngredient").value;
+    console.log(ingredient);
+    random.appendChild(document.createTextNode(ingredient));
+    ul.appendChild(random);
+}
+
+let addStep = () => {
+    let ul = document.getElementById("steps");
+    let random = document.createElement("li");
```

Comment:
random isn't a great/descriptive name, no need to change it, but naming variables well makes for better code!